xcode/Subconscious/Shared/AppTheme.swift

```diff
@@ -185,3 +185,16 @@ extension UIColor {
         UIColor(Color.accentColor)
     }
 }
+
+extension ShadowStyle {
+    static func brandShadowLg(
```

Comment:
Introduce the concept of shadow presets. In future we will want to systematize our shadows so as to have a coherent sense of distance and lighting.